Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG NEXUS_VERSION=3.19.0-SNAPSHOT
+
+FROM maven:3-jdk-8-alpine AS build
+ARG NEXUS_VERSION=3.19.0
+ARG NEXUS_BUILD=SNAPSHOT
+
+COPY . /nexus-repository-apk/
+RUN cd /nexus-repository-apk/; sed -i "s/3.19.0-02/${NEXUS_VERSION}-${NEXUS_BUILD}/g" pom.xml; \
```

Comment:
Make sure to check if this is the actual released version of `3.19.0-02`

Reply:
So we don't really need these manipulations any more and the 'sed' voodoo. See link in comment below for a simpler Dockerfile (from the nexus-format-archetype code base).

Reply:
in: 72661e1